static/app/components/asyncComponentSearchInput.tsx

```diff
@@ -158,4 +207,5 @@ const Form = styled('form')`
   position: relative;
 `;
 
-export default withRouter(AsyncComponentSearchInput);
+const AsyncComponentSearchInputWithRouter = withRouter(AsyncComponentSearchInput);
+export {AsyncComponentSearchInputWithRouter as AsyncComponentSearchInput};
```

Comment:
should we maybe export everything here?

```suggestion
export {
  AsyncComponentSearchInputProps,
  AsyncComponentSearchInputWithRouter as AsyncComponentSearchInput,
};
```

Reply:
Tbh, I'm not sure yet, we talked about not exporting types unless we need them at the TSC a few weeks back and for now, there doesn't seem to be a need to export them

Reply:
I agree but it seems the type is already being exported here https://github.com/getsentry/sentry/blob/9b0288dedbf40dbeab349bd569d896d0d9c8ba86/static/app/components/asyncComponentSearchInput.tsx#L47

Reply:
@priscilawebdev I gave this some though and I think I'm leaning towards keeping the exports close to the definitions - that way all of it is closer to the definition and it's less scrolling around to get to the export, wdyt?

Reply:
I changed it to your suggestion for this PR, but def something we should discuss w everyone :)